src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -0,0 +1,40 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import (
+    register_cli_argument,
+    CliArgumentType
+)
+
+from azure.cli.core.commands.parameters import (
+    resource_group_name_type,
+    location_type,
+    tags_type
+)
+
+from ._validators import validate_registry_name
+
+registry_name_type = CliArgumentType(
```

Comment:
>registry_name_type [](start = 0, length = 18)

registry_name and storage_account name should have completers for TAB completion purposes. You should be able to find examples in any of the major modules of how to do this. Also, you might keep the registry_name_type definition because you will typically have something like this:

```Python
registry_name_type = CliArgumenType(..., completer=...)

register_cli_argument('acr', 'registry_name', registry_name_type) # applies the base version generically
register_cli_argument('acr create', 'registry_name', registry_name_type, completer=None) # disables the completer for the create command
```